apps/web/pages/settings/my-account/profile.tsx

```diff
@@ -298,13 +300,7 @@ const ProfileView = () => {
             title={t("delete_account_modal_title")}
             description={t("confirm_delete_account_modal")}
             type="creation"
-            actionText={t("delete_my_account")}
-            actionProps={{
-              // @ts-expect-error data attributes aren't typed
-              "data-testid": "delete-account-confirm",
```

Comment:
Is this being used in a test?

Reply:
Yes but it also being passed into the button on line 328 so im not 100% sure what's happening here